index.html

```diff
@@ -115,84 +117,17 @@ <h3 class="course-heading">ASP.NET</h3>
         </div>
       </div>
       <a class="join-link" href="#">Join WebDev Academy 2023</a>
-      <a class="see-all-courses" href="">SEE ALL COURSES</a>
+      <div class="see-all-courses">
+        <a class="see-all-courses-link" href="">SEE ALL COURSES</a>
+      </div>
     </section>
 
     <!-- Program Teachers -->
     <section class="teachers">
       <h1 class="teachers-header">Program Teachers</h1>
```

Comment:
- [ ] You have done well in adhering to the correct hierarchy of `hx` elements :+1:. However, for semantic reasons, it is not recommended to use more than one `h1` element per page.